mne/io/eeglab/eeglab.py

```diff
@@ -119,6 +117,30 @@ def read_raw_eeglab(input_fname, montage=None, preload=False, eog=(),
         Path or instance of montage containing electrode positions.
         If None, sensor locations are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
+    eog : list | tuple | 'auto'
+        Names or indices of channels that should be designated EOG channels.
+        If 'auto', the channel names containing ``EOG`` or ``EYE`` are used.
+        Defaults to empty tuple.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated MISC
+        channels. Values should correspond to the electrodes in the vhdr file.
+        Default is ``()``.
```

Comment:
That's right we have little instances of tuples as defaults. Anyways it's a funny way of saying "nothing". I'm kind of used to the None.